lib/Conversion/StandardToLLVM/ConvertStandardToLLVM.cpp

```diff
@@ -883,7 +884,26 @@ struct CmpIOpLowering : public LLVMLegalizationPattern<CmpIOp> {
     rewriter.replaceOpWithNewOp<LLVM::ICmpOp>(
         op, lowering.convertType(cmpiOp.getResult()->getType()),
         rewriter.getI64IntegerAttr(
-            static_cast<int64_t>(convertCmpIPredicate(cmpiOp.getPredicate()))),
+            static_cast<int64_t>(convertCmpPredicate<LLVM::ICmpPredicate>(cmpiOp.getPredicate()))),
```

Comment:
I think this is going over 80 cols, did you clang-format your changes?

Reply:
Fixed.